.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,93 @@ assignees: ''
+- [ ] Completed the NIH [Information Security and Management Refresher](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Security Awareness for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Management for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the [dbGaP Code of Conduct](https://www.surveymonkey.com/r/HKXNYD7) quiz
+- [ ] Completed the [NIH Security Best Practices for (CAD) Subject to NIH (GDS) Policy](https://www.surveymonkey.com/r/FG3C63T) quiz
+- [ ] Digitally signed the [NIH GDS Policy](https://giwiki.gi.ucsc.edu/images/1/1b/NIH_GDS_Policy.pdf) document and gave it to your project manager
+- [ ] Created a GitHub account with your UCSC account email address
+- [ ] Installed [git-secrets](..#211-git-secrets)
+- [ ] Completed steps for [setting up a VPN client](..#911-setting-up-a-vpn-client) for `dev.gitlab` and `anvildev.gitlab`
+- [ ] Sent a request email to the organization administrator (cc: hiring supervisor) asking for:
+    - Creation of an AWS IAM user account
+    - Assign the `developer` role to the AWS IAM user account
+    - Assign the `developer` role to your GitHub user account
+    - Add your UCSC account to the `firecloud-cgl` group in Terra
+- [ ] Assigned ticket to the hiring supervisor
 
-- [ ] [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+## Hiring supervisor actions
 
-Install the following prerequisites:
+- [ ] Approved (by replying to) the request email sent by the onboarded developer to the organization administrator
+- [ ] Assigned ticket to the organization administrator
 
-- [ ] [git-secrets](..#211-git-secrets)
+## Organization administrator actions
 
-Request access to cloud infrastructure and tools:
+- [ ] Verified the authenticity of the approval
+- [ ] Verified the GitHub user account is associated with the onboarded developer's UCSC account email
+- [ ] Created an IAM user account for the onboarded developer
+- [ ] Added the `developer` role to the onboarded developer' IAM user account
+- [ ] Added the `developer` role to the onboarded developer's GitHub user account
+- [ ] Added the onboarded developer's UCSC account to the `firecloud-cgl` group in Terra
+- [ ] Assigned ticket to the onboarded developer
 
-- [ ] @theathorn: [UCSC-CGL Slack](ucsc-cgl.slack.com) through your Project Manager
- 
-- [ ] @hannes-ucsc: Push access to the Azul Repository on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+## Onboarded developer actions
 
-- [ ] @hannes-ucsc: Access to development CI/CD systems on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+- [ ] Sent a request email to the system administrator (cc: hiring supervisor) asking for:
+    - Creation of a GitLab user account
+    - Assign the `developer` role to the account
```

Comment:
```suggestion
- [ ] Creation of a GitLab user account
- [ ] Assign the `developer` role to the account
```

Reply:
The nested bulleted list for these items was intentional. It follows a checklist item to email the organization administrator and lists what should be requested (not performed at that time). There are correlating checklist items in the `Organization administrator actions` section for the requested actions.